### PR TITLE
508 compliance

### PR DESCRIPTION
**Pull request by ahse2**
Making the combobox accessible
expectation

### CODE REVIEW

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://6124020495333c00ad828e09--condescending-leavitt-66e607.netlify.app

File: packages/select/src/Select.js [hunk 1]
```diff
@@ -26,6 +27,15 @@ const components = {
       <span className="sr-only">Clear all selections</span>
     </ClearIndicator>
   ),
+  Option: (props) => {
+    const innerProps = {
+      ...props.innerProps,
+      role: 'option',
+      'aria-selected': props.isSelected,
```
Comment (chrishavekost) [inline packages/select/src/Select.js:33]: I think we should add some unit tests to check for the existence of these props and confirm accessibility of the options

File: packages/select/src/Select.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/prop-types */
```
Comment (chrishavekost) [inline packages/select/src/Select.js]: This should be removed, prop types should be added when needed

**Review by chrishavekost — CHANGES_REQUESTED** [review]

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://61255574de0fe916c3fc2828--condescending-leavitt-66e607.netlify.app

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://613ac3ee163b3f2b8580402d--condescending-leavitt-66e607.netlify.app

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://613aecfd082f9b70a5ab0251--condescending-leavitt-66e607.netlify.app

File: packages/select/src/Select.js (hunk 1 above)
Comment (LauRoxx) [reply packages/select/src/Select.js:33]: @chrishavekost @jordan-a-young 
Are Adrian's most recent commits sufficient?

**Review by jordan-a-young — APPROVED** [review]

File: packages/select/src/Select.js [hunk 3]
```diff
@@ -26,6 +27,15 @@ const components = {
       <span className="sr-only">Clear all selections</span>
     </ClearIndicator>
   ),
+  Option: (props) => {
+    const innerProps = {
+      ...props.innerProps,
+      role: 'option',
+      'aria-selected': props.isSelected,
+      name: 'option',
```
[the remark this answers is not in the document]
Comment (chrishavekost) [reply packages/select/src/Select.js]: @ahse2 confirmed that `name` is being used by screen readers

**Review by chrishavekost — APPROVED** [review]
lgtm, thanks!

**Pull request merged**